Assets/MixedRealityToolkit.Extensions/SceneTransitionService/Profiles/DefaultSceneTransitionServiceProfile.asset

```diff
@@ -23,5 +23,5 @@ MonoBehaviour:
   fadeTargets: 0
   cameraFaderType:
     reference: Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions.CameraFaderQuad,
-      Assembly-CSharp
+      Microsoft.MixedReality.Toolkit.Extensions.SceneTransitionService
```

Comment:
I think the Examples Hub profile needs this update as well

Reply:
It seems like Assembly-CSharp is not something that should exist in an MRTK asset. Will add code validation to check against that.